.travis/script.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 
-scripts_dir=$(dirname $(readlink -e "${BASH_SOURCE[0]}"))
-source "$scripts_dir/common";
+source "${0%/*}/setup.sh"
 
 onnx_dir="$PWD"
 
```

Comment:
Below you're installing onnx, which is already happening in install.sh. No need to do it again.

Reply:
++